tests/test_result.py

```diff
@@ -50,492 +44,6 @@ def test_uk_election_timetable():
     ] == datetime.timedelta(days=8)
 
 
-# https://election-timetable.democracyclub.org.uk/?election_date=2024-05-02
-
-
-@freeze_time("2024-04-04")
-def test_sopn_day(template_sorter, election_date_template_sorter):
```

Comment:
We talked about this earlier today.
Obviously it is bad that I am deleting a lot of tests here.
This is technical debt we will need to pay off later.
I've written up notes on this in https://app.asana.com/0/1204880927741389/1209880758255113/f